scripts/scripts.js

```diff
@@ -305,6 +322,13 @@ async function loadLazy(doc) {
   }
   loadCSS(`${window.hlx.codeBasePath}/styles/lazy-styles.css`);
   loadFonts();
+
+  // decorate links in main
+  decorateLinks(main);
+  // decorate links in header
+  decorateLinks(doc.querySelector('header'));
+  // decorate links in footer
+  decorateLinks(doc.querySelector('footer'));
```

Comment:
Is there a reason we couldn't combine it into a single `decorateLinks(doc)`?

Reply:
@fnhipster 
Going through the code I just had an impression that `main` has to be handled separately, for some reason (it has its dedicated `decorateMain()` function).
I combined those calls into a single one with `doc` as an argument.

Reply:
thanks!